docs/walkthrough.md

```diff
@@ -0,0 +1,551 @@
+This is the job of **content models**.
+
+That's what derivatives of `ContentBase` are. Anything that inherits from `ContentBase` will be available as a page type in the CMS.
+_There is no explicit registration of content models_; it just works.
+
+?> **Note:** We've named this model MyContent.
+In our companion <a href="https://github.com/onespacemedia/tiny-cms-project">tiny CMS project</a>, we've named it simply "Content".
+We use "MyContent" here to avoid confusion between the concept of a "content model", and our model that is named "Content".
+
+`introduction` is, of course, a standard Django field, which we'll use this later in our template.
+We can define no fields at all on this model! Just its existence as a non-abstract class inheriting from ContentBase will make it available in the page types.
+
+Now go to your admin and add a Page. You will be prompted to select a page type. Once you have selected "My content" as your page type, your page will appear with the `introduction` field all ready to fill out. Do that now, save the page, then go to the root URL on your website.
+
+Surprise! It's totally empty.
+Just as it doesn't have any assumptions about what your content looks like, it doesn't have any opinions on what the front end of your site should look like either.
+But the CMS is in fact rendering this view, and is making an educated guess as to what template it should use. It's falling back to your `base.html` at the moment, but that's not its first choice. Let's create a template called `content/mycontent.html`:
+
+```
+{% extends 'base.html' %}
+
+{% block main %} {# or whatever your main block is on your site :) #}
+  <h1>{{ pages.current.title }}</h1>
+
+  <p>{{ pages.current.content.introduction }}</p>
+{% endblock %}
+```
+
+Now reload your page. It has content! That's because if it hasn't been told to do anything else, it will look for a template at `<app_label>/<model_name>.html`. Convention over configuration ahoy!
+
+Where did `pages` come from? That would be the template context processor you added earlier.
+`pages.current` refers to the currently active Page.
+`pages.current.content` refers to the instance of your content model that is attached to the page.
+
+This is the very simplest example of rendering a page's content model on the front end.
+There's all sorts of other things we can do with content models, but if your model has some fields, and maybe some inlines, you don't _need_ to write any views, just a template.
+
+Did we (royal we) say inlines? We definitely did.
+
+## Lets add some admin inlines
+
+OK, so let's say you want to build your new content page out of an entirely arbitrary number of sections.
+That's fine too!
+We can add inlines to our change-page view in the admin.
+Here is what your model looks like.
+Pay special attention to the ForeignKey if nothing else - this is essential, and note that it is to `pages.Page` and _not_ your content model:
```

Comment:
Might be worth mentioning the implications of this - that your content section could be an inline on any `ContentBase` derivative.